### DISCUSSION TITLE
Add Mirror Mod for osu! standard

### DISCUSSION THREAD

**Discussion by Anthony1x** [ppy/osu#13088] [Ideas] [answered]
**Describe the new feature:** Title says it all. Flipping the map horizontally can not just help players remove mindblocks on some maps, it's also neat for left-handed players (like me :]) since there's a number of maps that are mapped with right-handed players in mind. It would remove "unwanted" difficulty for lefties.
Potential problem is that maps that are supposed to be hard for the right hand (top left corner) could easily be bypassed by using the mod.

**Proposal designs of the feature:** I'm not designer at all, but using the mania mirror mod icon (as seen on the website) and replacing the lines with circles should look pretty good. Multiplier should be 1x since it literally does nothing but flipping the map. 
![DKWYPGp](https://user-images.githubusercontent.com/42468660/69503426-d461ce00-0f19-11ea-8c73-1b96dfae7fad.png)

_Note that this is my first proposal so please tell me if I did something wrong_

**Comment by LastExceed** [comment]
bypassing difficulty using a mirror mod only works if the vast majority of a beatmap was built in way that favors one laterality over the other. Since lefties exist and we don't want either party to have a compeditive advantage over the other, a ranked x1.0 mirror mod should definitely be introduced just like in mania. 
So "difficulty bypass" is actually not a problem, but instead the solution to a problem (laterality disadvantage that is).

Another purpose of the mirror mod is bypassing mindblocks, which as far as I know exist in all modes so I would also be worth considering adding the mirror mod to ctb (for taiko i'd make it a skinning feature instead).

**Comment by smoogipoo** [comment, marked as answer]
Added in https://github.com/ppy/osu/pull/14016
